iothub/device/src/Transport/Amqp/AmqpLinkHelper.cs

```diff
@@ -39,8 +39,7 @@ TimeSpan timeout
                 LinkName = CommonResources.GetNewStringGuid(linkSuffix),
                 Role = false,
                 InitialDeliveryCount = 0,
-                Target = new Target() { Address = BuildLinkAddress(deviceIdentity, deviceTemplate, moduleTemplate) },
-                Source = new Source() { Address = deviceIdentity.IotHubConnectionString.DeviceId }
```

Comment:
Can we get confirmation if this is needed?

Reply:
just aligning with what master's doing.